src/main/java/com/investment/manager/service/InvestmentService.java

```diff
@@ -39,9 +66,12 @@ public InvestmentDTO get(Long id) throws NotFoundException {
 		throw new NotFoundException("Investment not found");
 	}
 
-	public List<InvestmentDTO> getAll() {
+	public Page<InvestmentDTO> getAll(int page, int size) {
+
+		@SuppressWarnings("static-access")
+		PageRequest pageRequest = PageRequest.of(page, size, Sort.DEFAULT_DIRECTION.ASC);
```

Comment:
PageRequest.of(page, size), se não passar um Sort.DIRECTION, ele ja usa por default o ASC, então nesse caso é desnecessário passar o terceiro parâmetro, aí pode remover até o @SupressWarning("static-access")